mmv1/third_party/terraform/services/netapp/resource_netapp_flex_storage_pool_test.go.erb

```diff
@@ -0,0 +1,200 @@
+<% autogen_exception -%>
```

Comment:
A few comments regarding this test file:
1. Nit: would you mind formatting the file? The indentation is a bit hard to read.
2. We would normally just include these tests in `resource_netapp_storage_pool_test.go.erb`, with the version guard around them. Is there a reason this wouldn't work?
3. I would suggest changing the strategy slightly. First, it might be good to have an example of the new fields being used in the documentation, which would most likely look like another (minimal) example file. Also, these tests shouldn't need to cover updating fields that are covered in `resource_netapp_storage_pool_test.go.erb`, so I would suggest focusing on minimal configs that test the switching behavior specifically.

Reply:
I have added the tests to a single file `resource_netapp_storage_pool_test.go.erb` which contains both beta and ga provider tests as it is done in other services and gated the tests with the macro to run in beta provider tests.